src/app/shared/idle-watcher.ts

```diff
@@ -6,7 +6,7 @@ export class IdleWatcherUtil {
    */
   public static generateSessionLength(): number {
 
-    let sessionLengthInMins: number = 90
+    let sessionLengthInMins: number = 1
```

Comment:
I think we want to revert this back to 90 so that it times out at 90 minutes

Reply:
I am leaving the time as they were until the testing is done by QA. Do you think we should change the numbers now ?

Reply:
I suppose that is okay if it is temporary. We will just need to remember to come back and change it before going to PROD while also making sure the changes are also applied to the `stage` branch too.

Reply:
Sure, I will make sure I change it. Just want to make QA job little easy, so will be in touch with QA.